### DISCUSSION TITLE
Installation guide or notes?

### DISCUSSION THREAD

**Discussion by seamonkey420** [kingo132/GPD-Win-Max-Hackintosh#3] [Q&A]
can someone who's done this on a Win Max post installation notes? i assume we extract the files to the EFI partition but would appreciate some guidance on the bios stuff.  tia!

**Comment by kingo132** [comment]
Try to read how to modify BIOS section in README.md. Some people report that you may get EFI working without modifying BIOS but I think it may encounter some problems. Maybe the DVMT is the one that must be modified, it should be a multiple of 32MB. The default value is 60MB.
